financial_assistant/README.md

```diff
@@ -0,0 +1,150 @@
+
+* **SambaStudio embedding model (Option 2)**: To increase inference speed, you can use a SambaStudio embedding model endpoint instead of using the default (CPU) Hugging Face embedding. Follow the instructions [here](../README.md#use-sambastudio-embedding-option-2) to set up your endpoint and environment variables. Then, in the [config file](./config.yaml), set the variable `type` in `embedding_model` to `"sambastudio"`, and set the configs `batch_size`, `coe` and `select_expert` according to your SambaStudio endpoint.
+
+## Windows requirements
+
+- If you are using Windows, make sure your system has Microsoft Visual C++ Redistributable installed. You can install it from [Microsoft Visual C++ Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/) and make sure to check all boxes regarding C++ section. (Compatible versions: 2015, 2017, 2019 or 2022)
+
+# Deploy the starter kit GUI
+
+We recommend that you run the starter kit in a virtual environment or use a container. We also recommend using Python >= 3.10 and < 3.12.
+
+## Option 1: Use a virtual environment
+
+If you want to use virtualenv or conda environment:
+
+1. Install and update pip.
+
+    ```bash
+    cd ai_starter_kit/enterprise_knowledge_retriever
+    python3 -m venv enterprise_knowledge_env
+    source enterprise_knowledge_env/bin/activate
+    pip  install  -r  requirements.txt
+    ```
+
+2. Run the following command:
+
+    > This kit can be deployed using either a simple LCECL implementation or a LangGraph implementation, both of which produce identical results. This demonstrates the flexibility of the kit and showcases how to leverage these two approaches in Rag applications.
+
+    - LCEL version
+
+        ```bash
+            streamlit run streamlit/app.py --browser.gatherUsageStats false 
+        ```
+
+    - LangGraph version
+
+        ```bash
+            streamlit run streamlit/langgraph_app.py --browser.gatherUsageStats false 
+        ```
+
+After deploying the starter kit you see the following user interface:
+
+![capture of enterprise_knowledge_retriever_demo](./docs/enterprise_knowledge_app.png)
+
+## Option 2: Deploy the starter kit in a Docker container 
+
+NOTE: If you are deploying the docker container in Windows be sure to open the docker desktop application. 
+
+To run the starter kit  with docker, run the following command:
+
+    docker-compose up --build
+
+You will be prompted to go to the link (http://localhost:8501/) in your browser where you will be greeted with the streamlit page as above.
+
+Here's a short video demonstrating docker deployment:
+
+https://github.com/sambanova/ai-starter-kit/assets/150964187/4f82e4aa-c9a9-45b4-961d-a4b369be5ec4
+
+## Further settings
+
+For the `SEC-EDGAR` functionalities, company name and email are used to form a user-agent of the form:
+USER_AGENT: ```<Company Name> <Email Address>```
+```
+# Your organization
+SEC_API_ORGANIZATION="<your organization>"
```

Comment:
If we are auto populating this, do we still need this in the readme?

Reply:
Yes, because we are only populating that for` prod_mode = True`.